apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -76,21 +77,29 @@ export default class TeacherHomepage extends Component {
     });
   }
 
+  dismissCensusBanner(onSuccess, onFailure) {
+    $.ajax({
+      url: `/api/v1/users/${this.props.teacherId}/dismiss_census_banner`,
+      type: "post",
+    }).done(onSuccess).fail(onFailure);
+  }
+
+  logDismissCensusBannerError = (xhr) => {
+    console.error(`Failed to dismiss future census banner! ${xhr.responseText}`);
```

Comment:
This should be fine, but in general we don't log much to console, and at least back when we supported IE9 this would have caused further errors because `console` was only available when the console was open.